src/data/buildings.ts

```diff
@@ -1,10 +1,15 @@
 import { Ability } from "../framework/types";
+import { ERROR_ON_MISSING_ABILITIES } from ".";
```

Comment:
This creates a circular import which has the potential to break in weird ways. Let's move this to somewhere else with no imports instead of using index (e.g., maybe a `config.ts` or sth)